packages/mobile/src/paymentRequest/IncomingPaymentRequestListItem.tsx

```diff
@@ -117,4 +109,6 @@ const styles = StyleSheet.create({
   },
 })
 
-export default withTranslation(Namespaces.paymentRequestFlow)(IncomingPaymentRequestListItem)
+export default connect<{}, DispatchProps, {}, RootState>(null, {
```

Comment:
In general, it's better not to `connect` items in a list but instead just connect the parent container and pass in props as necessary. Not a big deal though